contracts/lending/ttoken/strategies/compound/TTokenCompoundStrategy_1.sol

```diff
@@ -134,14 +138,12 @@ contract TTokenCompoundStrategy_1 is RolesMods, TTokenStrategy {
         uint16 storedRatio
     ) internal {
         // Calculate amount to rebalance
-        uint16 medianRatio =
-            (compoundStore().balanceRatioMax +
-                compoundStore().balanceRatioMin) / 2;
-        uint256 requiredBal =
-            NumbersLib.percent(
-                storedBal + compoundBal - amount,
-                medianRatio - storedRatio
-            );
+        uint16 medianRatio = (compoundStore().balanceRatioMax +
+            compoundStore().balanceRatioMin) / 2;
+        uint256 requiredBal = NumbersLib.percent(
+            storedBal + compoundBal - amount,
+            medianRatio - storedRatio
+        );
```

Comment:
I foresee merge conflicts with the fixes to the withdraw function in #411 
should probably rebase on top of